pkg/asset/agent/installconfig_test.go

```diff
@@ -108,7 +112,7 @@ platform:
 pullSecret: "{\"auths\":{\"example.com\":{\"auth\":\"c3VwZXItc2VjcmV0Cg==\"}}}"
 `,
 			expectedFound: false,
-			expectedError: `invalid install-config configuration: [platform.vsphere.ingressVIPs: Required value: must specify VIP for ingress, when VIP for API is set, Platform.VSphere.failureDomains.topology.folder: Required value: must specify a folder for agent-based installs]`,
+			expectedError: `invalid install-config configuration: [platform.vsphere.ingressVIPs: Required value: must specify VIP for ingress, when VIP for API is set, platform.vsphere.ingressVIPs: Required value: must specify at least one VIP for the Ingress, Platform.VSphere.failureDomains.topology.folder: Required value: must specify a folder for agent-based installs]`,
```

Comment:
I think there are a couple more test cases (i.e. taken from potential returned error) to cover here for vSphere: 

- `apiVIP missing`
- `both apiVIP and ingressVIP missing`
- `invalid value (not an IP) for apiVIP and ingressVIP`
- `when NOT using user-managed loadbalancer, apiVIP must be different than ingressVIP`.
- `when NOT using user-managed loadbalancer, apiVIP and ingressVIP must be in the machine network`.
- `If 2 API/Ingress VIPs are given, one must be ipv4 and other must be ipv6`
- `API/Ingress VIP must be of the same IP family with machine network's primary IP Family for dual-stack IPv4/IPv6`

I don't think these are really needed to fix the bug but it would be good to have these cases in a follow-up later 😄